application/src/main/kotlin/net/corda/v5/application/uniqueness/client/UniquenessCheckerClientService.kt

```diff
@@ -0,0 +1,55 @@
+package net.corda.v5.application.uniqueness.client
+
+import net.corda.data.uniqueness.UniquenessCheckResponse
+import net.corda.v5.base.annotations.DoNotImplement
+import java.time.Instant
+import java.util.concurrent.Future
+
+/**
+ * This service is used to queue a request for uniqueness checking.
+ * The request will be processed asynchronously and a [Future] will
+ * be returned that can be used to check the outcome.
+ *
+ * This service can be injected to either flows or other services.
+ *
+ * The actual request processing depends on which uniqueness service
+ * is invoked from the client.
+ *
+ * The API contains only one function that is used to queue up a request.
+ */
```

Comment:
```suggestion
/**
 * This service is used to request uniqueness checking. The request will be processed asynchronously
 * and a [Future] will be returned that can be used to check the outcome.
 *
 * This service can be injected to either flows or other services.
 */
```


 
 Think this captures what is needed at the interface level. Regarding the removed content:
 
 - Queueing is an implementation detail
 - It's self-evident that an interface could have multiple implementations; stating this doesn't add value to the interface description
 - Similarly, it's self evident the interface contains one function
```